[2022-01-21 10:43:52,656] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_basic_dag.dbt_run manual__2022-01-21T10:43:43.309234+00:00 [queued]>
[2022-01-21 10:43:52,666] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_basic_dag.dbt_run manual__2022-01-21T10:43:43.309234+00:00 [queued]>
[2022-01-21 10:43:52,666] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-21 10:43:52,667] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-21 10:43:52,667] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-21 10:43:52,678] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): dbt_run> on 2022-01-21 10:43:43.309234+00:00
[2022-01-21 10:43:52,683] {standard_task_runner.py:52} INFO - Started process 10624 to run task
[2022-01-21 10:43:52,695] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_basic_dag', 'dbt_run', 'manual__2022-01-21T10:43:43.309234+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dbt_basic.py', '--cfg-path', '/tmp/tmp8hptmzq_', '--error-file', '/tmp/tmp9ul4x0lh']
[2022-01-21 10:43:52,696] {standard_task_runner.py:77} INFO - Job 4: Subtask dbt_run
[2022-01-21 10:43:52,754] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_basic_dag.dbt_run manual__2022-01-21T10:43:43.309234+00:00 [running]> on host c20cfa89904c
[2022-01-21 10:43:52,811] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_basic_dag
AIRFLOW_CTX_TASK_ID=dbt_run
AIRFLOW_CTX_EXECUTION_DATE=2022-01-21T10:43:43.309234+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-21T10:43:43.309234+00:00
[2022-01-21 10:43:52,812] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-21 10:43:52,812] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt run --profiles-dir /home/***/.dbt --project-dir /opt/***/dbt']
[2022-01-21 10:43:52,823] {subprocess.py:85} INFO - Output:
[2022-01-21 10:43:55,950] {subprocess.py:89} INFO - Running with dbt=0.21.1
[2022-01-21 10:43:58,091] {subprocess.py:89} INFO - [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-01-21 10:43:58,091] {subprocess.py:89} INFO - There are 1 unused configuration paths:
[2022-01-21 10:43:58,091] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-01-21 10:43:58,091] {subprocess.py:89} INFO - 
[2022-01-21 10:43:58,104] {subprocess.py:89} INFO - Found 2 models, 4 tests, 0 snapshots, 0 analyses, 172 macros, 0 operations, 0 seed files, 0 sources, 0 exposures
[2022-01-21 10:43:58,106] {subprocess.py:89} INFO - 
[2022-01-21 10:43:59,574] {subprocess.py:89} INFO - 10:43:59 | Concurrency: 1 threads (target='dev')
[2022-01-21 10:43:59,574] {subprocess.py:89} INFO - 10:43:59 |
[2022-01-21 10:43:59,577] {subprocess.py:89} INFO - 10:43:59 | 1 of 2 START table model PUBLIC.my_first_dbt_model................... [RUN]
[2022-01-21 10:44:01,447] {subprocess.py:89} INFO - 10:44:01 | 1 of 2 OK created table model PUBLIC.my_first_dbt_model.............. [[32mSUCCESS 1[0m in 1.87s]
[2022-01-21 10:44:01,448] {subprocess.py:89} INFO - 10:44:01 | 2 of 2 START table model PUBLIC.my_second_dbt_model.................. [RUN]
[2022-01-21 10:44:02,857] {subprocess.py:89} INFO - 10:44:02 | 2 of 2 OK created table model PUBLIC.my_second_dbt_model............. [[32mSUCCESS 1[0m in 1.41s]
[2022-01-21 10:44:02,908] {subprocess.py:89} INFO - 10:44:02 |
[2022-01-21 10:44:02,908] {subprocess.py:89} INFO - 10:44:02 | Finished running 2 table models in 4.79s.
[2022-01-21 10:44:02,909] {subprocess.py:89} INFO - 
[2022-01-21 10:44:02,910] {subprocess.py:89} INFO - [32mCompleted successfully[0m
[2022-01-21 10:44:02,910] {subprocess.py:89} INFO - 
[2022-01-21 10:44:02,910] {subprocess.py:89} INFO - Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2022-01-21 10:44:04,333] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-21 10:44:04,371] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_basic_dag, task_id=dbt_run, execution_date=20220121T104343, start_date=20220121T104352, end_date=20220121T104404
[2022-01-21 10:44:04,395] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-21 10:44:04,433] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
